__tests__/graphql.test.js

```diff
@@ -0,0 +1,132 @@
+import { graphql } from 'graphql';
+import schema from '../server/graphql';
+
+jest.mock('../server/graphql/jsonData', () => ({
```

Comment:
It might seem that way, but my intention was to test the implementation of the resolvers, which, unfortunately I can't do without calling graphql.